__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const { topicData, articleData, userData, commentData } = require('../db/data/test-data/index');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe('GET /api/topics', () => {
+    test('returns array of objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) => {
+            const topics = JSON.parse(response.text).topics;
```

Comment:
This is not how we should be getting topics. JSON.parse(...) should not need to be used to access the object we are sending. 

I would recommend revisiting the notes for this if you are unsure:
https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=supertest,step=intro